src/pages/ProductListing/ProductListing.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import Filter from '../../components/filter/Filter';
+import Product from '../../components/product/Product';
+import './ProductListing.css';
+
+const ProductListing = () => {
+
+    const [products, setProducts] = useState([]);
+
+    const fetchProducts = async () => {
+        try {
+            const products = await axios.get('/api/products');
+            setProducts([...products.data.products]);
```

Comment:
Is there any specific reason to destructure the products and the passing to state, coz you are getting already an array of products? So you can directly put them in.

Reply:
No, there is no such reason, we can use either ways